main.go

```diff
@@ -211,6 +213,15 @@ func getPacketConfig(providerConfig string) (packet.Config, error) {
 		config.APIServerPort = packet.DefaultAPIServerPort
 	}
 
+	config.BGPNodeSelector = rawConfig.BGPNodeSelector
+	if v := os.Getenv(envVarBGPNodeSelector); v != "" {
+		config.BGPNodeSelector = v
+	}
```

Comment:
Can we have a CLI flag option as well, like most of the ones above?

Reply:
Can you elaborate how to add it? I cannot figure it out from the code. Passing `-h` to the binary is not doing anything either, so I cannot even check what flags are available. README.md doesn't mention any CLI flag other than `-v` either :confused:

Reply:
Huh. Never mind, I thought we had added all of that here, but it is in a different project. Ignore this.